nengo/dists.py

```diff
@@ -335,6 +335,25 @@ def cdf(self, x):
             sq_x < 1., betainc(self.m / 2.0, self.n / 2.0, sq_x),
             np.ones_like(x))
 
+    def ppf(self, y):
+        """Inverse cumulative distribution function.
```

Comment:
Say what PPF stands for here, to help people understand why it's called "ppf" (I'd never heard the term before scipy, and I honestly can't remember what it is right now, but it's in the scipy docs).